src/deploy/NVA_build/Base.Dockerfile

```diff
@@ -27,6 +27,7 @@ ARG BUILD_S3SELECT=1
 ARG BUILD_S3SELECT_PARQUET=0
 #Clone S3Select and its two submodules, but only if BUILD_S3SELECT=1.
 RUN ./src/deploy/NVA_build/clone_s3select_submodules.sh
+RUN ln -s /lib64/libboost_thread.so.1.* /lib64/libboost_thread.so.link
```

Comment:
I don't think that using `ln -s` with a wildcard is ok...
I think we should explicitly do:
```
RUN sed -i -e 's|1.75.0|1.66.0|g' $REMOTE_SOURCE_DIR/noobaa-core/app/src/native/s3select/s3select.gyp
```
When the RHEL version is 8.

Reply:
Changed to symlink from 75 to 66 specifically.
Some more details in gyp comment.